dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/controller/DPPPTConfigController.java

```diff
@@ -49,14 +50,24 @@ public DPPPTConfigController() {
 
 	@CrossOrigin(origins = { "https://editor.swagger.io" })
 	@GetMapping(value = "")
+	@Documentation(description = "Echo endpoint",
+	responses = {"200 => Hello from DP3T Config WS"})
 	public @ResponseBody String hello() {
 		return "Hello from DP3T Config WS";
 	}
 
 	@CrossOrigin(origins = { "https://editor.swagger.io" })
 	@GetMapping(value = "/config")
-	public @ResponseBody ResponseEntity<ConfigResponse> getConfig(@RequestParam(required = true) String appversion,
-			@RequestParam(required = true) String osversion, @RequestParam(required = true) String buildnr) {
+	@Documentation(description="Read latest configuration and messages, depending on the version of the phone and the" +
+			" app.",
+	responses = {"200 => ConfigResponse structure with eventual notifications and epidemic parameters"})
+	public @ResponseBody ResponseEntity<ConfigResponse> getConfig(
+			@RequestParam
```

Comment:
```suggestion
                         @Documentation(description = "Version of the App installed", example = "ios-1.0.7") @RequestParam String appversion,
			@RequestParam
```

Reply:
the appversion was removed?

Reply:
It is in fact not used in the code... Only the `buildnr` is used, this is why I removed it. Does it make sense to keep it around even if it's not used?

Reply:
it's currently not used, since the client apps had a problem with the update note. But it will be used later on. Also, we need to have the option to change configuration according to these three criteria at any time (e.g. there is a critical vulnerability in version 1.0.x we need to have the option for a kill switch).